### ISSUE TITLE
Edges styling

### ISSUE THREAD

**Issue by jaintrai11** [neo4j-contrib/neovis.js#238]
![image](https://user-images.githubusercontent.com/26081328/153138537-a8f9eba1-9246-48b1-898a-f164f20bc7dd.png)

How to apply styling on Edges like font-size, etc

**Comment by LukasRasmussen** [comment]
```
neovis.registerOnEvent('completed', () => {
  neovis.edges.updateOnly( neovis.edges.get().filter(<filter-function>).map(edge=> ({...edge, "font": {"size":"16"} })) );
});
```
No need to use .filter() if you dont want to but it's nice if you want to style one particular type of edge.

**Comment by thebestnom** [comment]
@LukasRasmussen Not really needed anymore in 2.x you can just use the font directly in the config

**Comment by LukasRasmussen** [comment]
@thebestnom True! Considering you just put 2.x as latest everyone should use that from now on. I just thought that since 1.6 was latest before, people likely use that. Especially those who has asked styling questions before like 4 days ago. I commented this on a few issues, hopefully to some use to someone.
